site/config.toml

```diff
@@ -132,6 +142,14 @@ pygmentsStyle = "manni"
     button = "Start Extending"
     icon = "/img/extend-icon.svg"
     draft = false
+    
+  [[params.intro.item]]
+    title = "Apps Marketplace"
+    description = "Join the hundreds of contributors and submit code directly to the Mattermost open-source project."
+    url = "apps/readme"
```

Comment:
Are we ready for that kind of advertisement yet? Plugins are a big feature and yet places under `extend`. Should we also move `Apps` ther?

Reply:
Oh definitely not - sorry, this is still in draft and that's still placeholder text from one of the other sections - and yes, definitely, it can be moved around.